src/test/java/uk/gov/hmcts/dts/fact/controllers/admin/AdminCourtGeneralInfoControllerTest.java

```diff
@@ -0,0 +1,92 @@
+package uk.gov.hmcts.dts.fact.controllers.admin;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+import uk.gov.hmcts.dts.fact.exception.NotFoundException;
+import uk.gov.hmcts.dts.fact.model.admin.CourtGeneralInfo;
+import uk.gov.hmcts.dts.fact.services.admin.AdminCourtGeneralInfoService;
+
+import static org.mockito.Mockito.when;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.put;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@WebMvcTest(AdminCourtGeneralInfoController.class)
+@AutoConfigureMockMvc(addFilters = false)
+public class AdminCourtGeneralInfoControllerTest {
+    private static final String BASE_PATH = "/courts/";
+    private static final String CHILD_PATH = "/generalInfo";
+    private static final String TEST_SLUG = "unknownSlug";
+    private static final CourtGeneralInfo COURT_GENERAL_INFO = new CourtGeneralInfo(
+        true,
+        false,
+        "English info",
+        "Welsh info",
+        "English alert",
+        "Welsh alert"
+    );
+
+    private static String courtGeneralInfoJson;
+
+    @Autowired
+    private transient MockMvc mockMvc;
+
+    @MockBean
+    private AdminCourtGeneralInfoService adminService;
+
+    @BeforeAll
+    static void setUp() throws JsonProcessingException {
+        courtGeneralInfoJson = new ObjectMapper().writeValueAsString(COURT_GENERAL_INFO);
```

Comment:
Same thing with this one, can we initialise it on line 38? No worries if it's related to the JsonProcessingException which may be thrown